### PR TITLE
Unlock email notification

### PR DESCRIPTION
**Pull request by Aiky30**

### CODE REVIEW

File: djangocms_version_locking/emails.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+def notify_version_author_version_unlocked(version, unlocking_user):
+    # If the unlocking user is the current author, don't send a notification email
+    if version.created_by == unlocking_user:
+        return
+
+    site = get_current_site()
+    recipients = [version.created_by]
+    subject = "[Django CMS] ({site_name}) {page_title} - {description}".format(
+        site_name=site.name,
+        page_title=version.content,
```
Comment (chaosk) [inline djangocms_version_locking/emails.py]: Doesn't have to be a Page, so `page_title` is misleading

File: djangocms_version_locking/emails.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        description=_("Unlocked"),
+    )
+    version_url = reverse(
```
Comment (chaosk) [inline djangocms_version_locking/emails.py]: `get_object_preview_url(version.content)`

File: djangocms_version_locking/emails.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        'admin:cms_placeholder_render_object_preview',
+        args=(version.content_type_id, version.object_id),
+    )
+    version_url = get_absolute_url(version_url)
```
Comment (chaosk) [inline djangocms_version_locking/emails.py]: Perhaps pass the request to this function and call `request.build_absolute_uri(version_url)`?

Comment (Aiky30) [reply djangocms_version_locking/emails.py]: I took this helper from Moderation. Surely we should be consistent?

File: djangocms_version_locking/emails.py (hunk 2 above)
Comment (Aiky30) [reply djangocms_version_locking/emails.py]: Agreed, I took the example from Versioning compare i.e. how to see an unpublished version which is what Compare does. We may need to update the compare view in Versioning.

File: djangocms_version_locking/emails.py (hunk 1 above)
Comment (Aiky30) [reply djangocms_version_locking/emails.py]: Agreed.

**Pull request merged**